### DISCUSSION TITLE
Lumia 2520 - device request

### DISCUSSION THREAD

**Discussion by timmynana** [RedGreenBlue09/WFAv7_Installer#4] [Ideas]
Hi. I appreciate that this device was originally Windows RT 8.1 rather than Windows Phone, but would it be possible to add this to the list of working devices? It is a snapdragon 800 so is ARM 32 bit. Thanks.

**Comment by RedGreenBlue09** [comment]
Thanks for your request. Unfortunately, I was unable to find an instruction to install Windows 10 ARMv7 on the Lumia 2520. The closest one is Surface RT/2 which is ran on Tegra SOCs. Also I don't have that device to test.

**Reply by timmynana** [reply]
Thanks, I also cannot find any instructions. Whilst similar to Surface RT/2, the Lumia is a Snapdragon 800 rather than a Tegra, so it closer to the other devices capable under this installer. If someone can confirm how to dump any necessary documents please feel free to ask and I will help if possible.

**Reply by timmynana** [reply]
In fact, the specs are very very similar to the Lumia 1520 which is supported.

**Reply by RedGreenBlue09** [reply]
> In fact, the specs are very very similar to the Lumia 1520 which is supported.

Yeah it's the same SOC but with a very different bootloader, I didn't figured the way to unlock the device to install Windows 10 ARM.
